source/digits_hits/src/GateJPETGammaPolarizationActor.cc

```diff
@@ -418,8 +418,13 @@ void GateJPETGammaPolarizationActor::GetThetaAndPhi(const G4ThreeVector& k0, con
 
 		G4double k_x = k.dot(versorX);
 		G4double k_y = k.dot(versorY);
+		G4double sinTheta = sin(theta);
+		if(sinTheta == 0){
```

Comment:
Definitely you should add some comment here to explain why. It is not obvious

Reply:
Done. I am not going to use asserts here - I do not have an idea where place them now.  Problem with asserts is for next PR in my opinion.

Reply:
Ok. I will explain you later.